src/documents/IoCAPI.md

```diff
@@ -0,0 +1,133 @@
+
+### fluid.resolveContext (context, that)
+
+Another method for enthusiasts. Given a particular IoC [Context](Contexts.md), determine which (if any) component in the tree it refers to, from the point
+of view of component `that`.
+
+* `context {String}` The context to be resolved. For example, in an [IoC reference](IoCReferences.md) of the form `"{myContext}.myPath1.myPath2"`, the context is `myContext`.
+* `that {Component}` The component from whose point of view the context is to be resolved
+* Returns: `{Component|None}` Returns the component to which the context refers, or `undefined` if the reference cannot be resolved.
+
+### fluid.parseContextReference(reference) {
+
+Parse an [IoC reference](IoCReferences.md) of the form `"{myContext}.myPath1.myPath2"` into an object form. In this case, for example, the output would be 
+
+```json
+{
+    context: "myContext", 
+    path: "myPath1.myPath2"
+}
+```
+
+From here it is a simple matter to resolve them in a DIY fashion with reference to the APIs `fluid.resolveContext` and `fluid.getForComponent` or `fluid.get`.
+
+* `reference {String}` An [IoC reference](IoCReferences.md) expressed as a String
+* Returns: `{Object}` The parsed form of the reference, with the following fields:
+    * `context {String}` The context portion of the IoC reference
+    * `path {String}` The path portion of the IoC reference
+
+### fluid.globalInstantiator
+
+This path holds the global ***instantiator*** which holds all the records for Infusion's IoC system. Whilst any methods 
+on this object should not be called by applications, there are many entries in here that can aid debugging, especially
+the members `instantiator.pathToComponent` which holds a mapping for every instantiated component from its global path
+in the component tree, and `instantiator.idToShadow` which holds a mapping from every component's `id` to its ***shadow record***
+which holds many useful pieces of bookkeeping, including the `mergeOptions` structure which can be used to inspect the details
+of the options merging process resulting in the component's final options.
+
+### fluid.rootComponent
+
+Holds the global root component for the global instantiator's component tree.
+
+
+## Nexus API methods 
+
+### fluid.construct(path, options[, instantiator])
+
+Construct a component with the supplied options at the specified path in the component tree. The parent path of the location must already be a component.
+
+* `path {String|Array of String}` Path where the new component is to be constructed, represented as a string or array of segments
+* `options {Object}` Top-level options supplied to the component - must at the very least include a field `type` holding the component's type. Note that these are expressed
+in the currently hypothetical [post-FLUID-5750](https://issues.fluidproject.org/browse/FLUID-5750) format with `type` alongside the component's options rather than at
```

Comment:
It is very disconcerting to read "currently hypothetical" in API documentation. Perhaps leave out that part, and just describe the format?